app/src/main/res/values/strings.xml

```diff
@@ -100,6 +100,16 @@
     <string name="notification_title">%1$s will start soon</string>
     <string name="notification_message">%1$s ~ %2$s @ %3$s</string>
 
+    <!-- questionnaire -->
+    <string name="questionnaire">Questionnaire</string>
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire appears to be unused [UnusedResources]